tests/models/encoder_decoders/wikitables_semantic_parser_test.py

```diff
@@ -25,6 +25,105 @@ def setUp(self):
     def test_model_can_train_save_and_load(self):
         self.ensure_model_can_train_save_and_load(self.param_file)
 
+    def test_get_neighbor_indices(self):
+        worlds, num_entities = self.get_fake_worlds()
+        tensor = Variable(torch.LongTensor([]))
+
+        neighbor_indexes = self.model._get_neighbor_indices(worlds, num_entities, tensor)
+
+        # Checks for the correct shape meaning dimension 2 has size num_neighbors,
+        # padding of -1 is used, and correct neighbor indices.
+        assert torch.equal(neighbor_indexes.data, torch.LongTensor([[[2, 3],
```

Comment:
Why does this have shape (1, 6, 2)?  I would have thought it would be (1, 4, 2).  Ah, it's because you have a `sum()` down below in `get_fake_worlds`, where it should be a `max()`.